test/000_fullchain.test.ts

```diff
@@ -213,17 +219,17 @@ describe('Integration tests', function () {
         );
         const taskPrice = appPrice + datasetPrice + workerpoolPrice;
         const schedulerStakePerTask = schedulerStakePerDeal / volume;
-        const workerRewardPerTask = await iexecWrapper.computeWorkerRewardPerTask(
+        const workersRewardPerTask = await iexecWrapper.computeWorkersRewardPerTask(
             dealId,
             PocoMode.CLASSIC,
         );
-        const schedulerRewardPerTask = workerpoolPrice - workerRewardPerTask;
+        const schedulerRewardPerTask = workerpoolPrice - workersRewardPerTask;
         // Save frozens
 
         const accounts = [requester, scheduler, appProvider, datasetProvider, ...workers];
-        const accountsInitialFrozens = await getInitialFrozens(accounts);
+        const accountsInitialFrozens = await getInitialFrozens(iexecPoco, accounts);
         // Finalize each task and check balance changes.
-        const workerStakePerTask = await iexecPoco
+        const workerStake = await iexecPoco
```

Comment:
I find it clearer with the postfix:
```suggestion
        const workerStakePerTask = await iexecPoco
```